docs/source/mtia.rst

```diff
@@ -0,0 +1,20 @@
+torch.mtia
```

Comment:
nit: can you add a bit more doc here to mention that this device is implemented our of tree and only the API is defined here.